docs/pages/reference/authentication.mdx

```diff
@@ -121,6 +121,49 @@ $ tctl create -f cap.yaml
 </TabItem>
 </Tabs>
 
+### Local User Locks
+
+A lock on attempting logins is placed for 20 minutes if within a 30 minute window 
+a local user has:
+- 5 failed login attempts or
+- 3 failed password resets
+
+After the lock has expired the user may attempt to login again.
+
+Overriding the lock is available to users with rights to maintain `user` resources,
+available in the builtin `editor` role.  To turn off a lock update the user entry
+to disable the lock. Here are steps to complete that.
+
+Retrieve the user entry so you can edit the status:
+
+```code
+$ tctl get users/jeff > jeff.yaml`
```

Comment:
Can/should we use `tctl edit` here?

Reply:
For v13 and v12 we can.

Reply:
When I last tried it wasn't allowing it in v12. Is there an update coming?

Reply:
No, `tctl edit` has just only been supported for v12+

Reply:
This issue #21598

Reply:
Ah, right.

Reply:
Let's go ahead with this and can update if that changes